imagely.bin.js

```diff
@@ -39,13 +79,19 @@ if (options.log) {
 else if (options.batch) {
 	callback = function() {
 		if (this.jsonIndex < this.json.length) {
-			var name = this.destination.split('.').slice(0, -1).pop();
-			var uniqueName = this.destination.replace(name, name + this.jsonIndex);
-			var html = this.setWindowData(this.originalHtmlString, JSON.stringify(this.json[this.jsonIndex]));
+			var json = this.json[this.jsonIndex];
+			var uniqueName;
+			var name = this.destination.split('/').slice(-1).pop();
```

Comment:
This isn't a platform-compatible way to parse filepaths (eg. on Windows the delimiter isn't `/`). Use the node `path` module instead (eg. `path.basename()`).